server/src/server.js

```diff
@@ -0,0 +1,34 @@
+import express from 'express';
+import morgan from 'morgan';
+import cors from 'cors';
+import bodyParser from 'body-parser';
+import config from './config';
+import { connect } from './utils/db';
+import { statusHandler } from './utils/errorHandler';
+import { publicRouter, protectedRouter } from './resources/router';
+
+export const app = express();
+
+app.use(cors());
+app.use(express.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(morgan('dev'));
+app.use(express.json());
+app.use(bodyParser.urlencoded({ extended: false }));

```

Comment:
It looks like there's some repeated code here; there isn't a purpose for this, is there?

Reply:
Nope. We'll need to remove it. That looks like it got through a code review